package.json

```diff
@@ -91,7 +91,7 @@
   },
   "devDependencies": {
     "@edx/browserslist-config": "1.0.0",
-    "@edx/frontend-build": "^12.9.0-alpha.1",
+    "@openedx/frontend-build": "13.1.0-alpha.2",
```

Comment:
This was intentional as v14 contains `jest` v29 upgrade & this PR focuses on `@edx/paragon` & `@edx/frontend-build` to `openedx` namespace switch. There is another [PR (currently drafted)](https://github.com/openedx/frontend-app-admin-portal/pull/1202/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R94) for `jest` v29 upgrade, containing upgrade to `frontend-build` v14 along with respective `edx` packages, which will be updated and ready for review once this PR is merged

Reply:
I'm not really following why this work needs to be split into 2 separate PRs at this point. Why can't the upgrades from the [draft PR](https://github.com/openedx/frontend-app-admin-portal/pull/1202/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R94) just be migrated to this PR since they are so related anyways? IMHO, I would have considered Jest v29 upgrade (and its other Jest-related package upgrades) as part of the original scope of migrating to latest version of `@openedx/frontend-build` in this PR directly; personally don't really see why this work needs to be split across 2 PRs.

Either way, I'm not going to push back further if you feel the incremental upgrade and separate upgrade is truly necessary versus bringing the other upgrades into this PR (given Jest upgrade, etc. actually _is_ related to the scope of migrating to `@openedx/frontend-build`, IMHO).

Reply:
Thank you for your feedback. My rationale behind splitting the work into 2 different PRs was based on 
1. Scope and Complexity Management: The primary focus of the current PR is the namespace switch to `@openedx` for `@edx/paragon` and `@edx/frontend-build`. This was scoped as a standalone epic to ensure a clean transition without introducing additional variables. Also the `frontend-build` `v14` brings in several major package upgrades further adding to the complexity which when handled separately could ensure a more focused and manageable review process. 
2. Project Timeline and Dependencies: When the PR was initially created, `frontend-build `v14 was not yet available. It was intended to address the namespace switch only. The subsequent release of `frontend-build `v14 necessitated additional work, which was scoped into a separate Jest v29 epic
3. Given the significant number of file changes and potential issues arising from multiple major upgrades, splitting the work allows for easier tracking and isolation of issues, also incremental approach reduces the risk of introducing widespread breaking changes and simplifies troubleshooting if things go south.
4. The upgrades correspond to different epics, one focused on the namespace switch and the other on upgrading Jest and related packages. Keeping them separate maintains alignment with the objectives.

I understand the perspective that combining the upgrades might seem more efficient.  I have decided to combine the two considering the review process duration and the fact that the change itself, while widespread, is not overly complicated, it will ultimately expedite the merge process and reduce the overall time required for reviews by having single review for both features.